core/src/main/java/com/sequenceiq/cloudbreak/service/stack/connector/aws/CloudFormationStackStatusChecker.java

```diff
@@ -90,6 +91,11 @@ public boolean exitPolling(CloudFormationStackPollerObject cloudFormationStackPo
         }
     }
 
+    @Override
```

Comment:
Maybe it would be better to create an Abstract class and extend the checker tasks from it to avoid code duplication where it's possible.